webextensions/api/devtools.json

```
@@ -314,7 +572,48 @@
               }
             }
           },
+          "Button": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": true
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "16"
+                },
+                "safari_ios": {
+                  "version_added": false
+                }
+              }
+            }
+          },
           "ElementsPanel": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/devtools/panels/ElementsPanel",
+              "support": {
+                "chrome": {
+                  "version_added": true
```

Already mentioned in [this commit before Chrome 13](https://github.com/chromium/chromium/commit/86f8bcbe064f1bb59fff9ff2caba239a1895aca9#diff-418810b3ccbf5475e996fb8fff2483b6d3a08ceedf639cf6e590a3ad2d8262eaR38) when `devtools.panel` was still experimental:

```suggestion
                  "version_added": "18"
```